src/taco-utils/tacoPackageLoader.ts

```diff
@@ -322,7 +322,7 @@ module TacoUtility {
         private static createTacoPackageInstallRequest(packageKey: string, dependencyConfigPath: string, logLevel: InstallLogLevel): IPackageInstallRequest {
             if (fs.existsSync(dependencyConfigPath)) {
                 try {
-                    var dependencyLookup: any = require(dependencyConfigPath);
+                    var dependencyLookup: any = JSON.parse(<any> fs.readFileSync(dependencyConfigPath));
```

Comment:
I think it'd be safer to pass an encoding, or assume that you get back a buffer. According to this: https://github.com/nodejs/node-v0.x-archive/issues/186 that call to readFileSync will return a buffer, and that matches our .d.ts file:
    export function readFileSync(filename: string, encoding: string): string;
    export function readFileSync(filename: string, options: { encoding: string; flag?: string; }): string;
    export function readFileSync(filename: string, options?: { flag?: string; }): Buffer;